sim/pokemon.js

```diff
@@ -1178,7 +1172,7 @@ class Pokemon {
 				break;
 			}
 
-			this.battle.singleEvent('Use', item, this.itemData, this, source, sourceEffect);
+			this.battle.singleEvent('Use', item, this.itemData, this, source, this.battle.effect);
```

Comment:
I definitely don't agree with this instead of manually specifying `sourceEffect`.

Reply:
Drop `itemName` if you must, but leave `sourceEffect` alone.

Reply:
I don't understand what you mean by manually specifying `sourceEffect`. Do you want to leave it as an unused parameter?

(Sorry for the delayed response; I suck at time management, so I tend to acquire a backlog.)

Reply:
Yes, I want to leave it as an unused parameter.

Reply:
(Specifically: to keep our API surface consistent for mods.)